docs(README.md): fix setup section & add informations about global configuration file (#210)
RobotecAI/rai: README.md

### 3. Setting up vendors

> RAI is fully vendor-agnostic, however the beta development work currently utilizes OpenAI models (mostly).

Reviewer: ```suggestion
```

Reviewer: This seems to be covered by sentence below and note

> Most of the RAI modules, will use the configuration from [config.toml](./config.toml) file in which you can set up your vendor of choice.

Reviewer: ```suggestion
RAI aims to be vendor-agnostic. You can use the configuration in [config.toml](./config.toml) to set up your vendor of choice for most RAI modules.
```
